include/ignition/math/Vector2.hh

```diff
@@ -128,8 +128,8 @@ namespace ignition
       /// \return the result
       public: Vector2 Round()
       {
-        this->data[0] = nearbyint(this->data[0]);
-        this->data[1] = nearbyint(this->data[1]);
+        this->data[0] = static_cast<T>(nearbyint(this->data[0]));
```

Comment:
The https://en.cppreference.com/w/cpp/numeric/math/nearbyint function returns the same type as its argument, so I'm suspicious of the need for a cast here.  Is the compiler selecting the wrong overload?  Maybe we should be calling `std::nearbyint` instead of using ADL or relying on the C-global namespace lookup?  It also seems like this file is missing the `#include` statement for `<cmath>`, which could also be the problem.